sql/core/src/test/scala/org/apache/spark/sql/ResolveDefaultColumnsSuite.scala

```diff
@@ -182,14 +182,14 @@ class ResolveDefaultColumnsSuite extends QueryTest with SharedSparkSession {
             "actualType" -> "\"STRING\""))
         checkError(
           exception = intercept[AnalysisException] {
-            sql("create table demos.test_ts_other (a timestamp default '2022-01-02') using parquet")
```

Comment:
The original test seems incorrect as the expression `a timestamp default '2022-01-02'` is actually valid, which meets the requirement of fitting within the narrower typ